frontend/src/components/shared/FullScreenWrapper/styled-components.ts

```diff
@@ -24,24 +24,43 @@ export interface StyledFullScreenButtonProps {
 export const StyledFullScreenButton = styled.button<
   StyledFullScreenButtonProps
 >(({ isExpanded, theme }) => {
+  const variableProps = isExpanded
+    ? {
+        right: "1rem",
+        top: "0.5rem",
+        backgroundColor: "transparent",
+      }
+    : {
+        right: "-3.0rem",
+        top: "-0.375rem",
+        opacity: 0,
+        transform: "scale(0)",
+        backgroundColor: theme.colors.lightenedBg05,
+      }
```

Comment:
![BE39AC77-5FA0-4EF6-9478-A48CF9D56007](https://user-images.githubusercontent.com/69432/129428070-88e4f599-9ecc-4a15-945f-d7dcae2b5551.png)
Something is not right here. Let me know if you want me to investigate why.